### PR TITLE
Update routes_test.rb

### PR DESCRIPTION
**Pull request by paulinagadea**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- `rake test`
* [ ] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [ ] pull request are descriptively named
* [ ] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

File: test/integration/routes_test.rb [hunk 1]
```diff
@@ -5,7 +5,12 @@
 class RoutesTest < ActionDispatch::IntegrationTest
   fixtures :all
 
-  test "test spectrums choose route" do
+   "test spectrums choose route" do
```
Comment (Tlazypanda) [inline test/integration/routes_test.rb:8]: Hey @PaulinaGadea looks like you have accidentally removed the test word here which is why the tests are failing can you add it back? Rest all looks great :100:  Thanks :v:

Comment (PaulinaGadea) [reply test/integration/routes_test.rb]: Hi @Tlazypanda, thank you so much! I hope it's okay now

**Comment by PaulinaGadea** [comment]
Hi @TildaDares! I finished my first issue and I was wondering if you could check that everything is okay, thanks

**Comment by Tlazypanda** [comment]
Hey @PaulinaGadea sorry to inform you but this issue has been closed now, can I interest you in another issue to take up?

**Comment by TildaDares** [comment]
Thanks for working on this issue @paulinagadea but this issue has been resolved in #799. If you'd like to be assigned to another issue, you can leave a comment on https://github.com/publiclab/plots2/issues/10153 and we'll assign one to you as soon as it's available.

**Pull request closed without merging**